assets/js/frontend/checkout.js

```diff
@@ -583,6 +583,7 @@ jQuery( function( $ ) {
 						$form.before( code );
 						$form.slideUp();
 
+						$( document.body ).trigger( 'applied_coupon', [ data.coupon_code ] );
```

Comment:
I think we should call it something other than `applied_coupon`, maybe `applied_coupon_in_checkout` to prevent confusion with `applied_coupon` trigger in cart.js.
I think in many cases, developers would link the same function, but using the same trigger name would cause unexpected/untested behavior in older usages.

Reply:
That's actually a great point, changing it now!